Sorts/BogoSort.js

```diff
@@ -1,5 +1,7 @@
 /**
  * Checks whether the given array is sorted in ascending order.
+ * @param {number[]} array The array to be checked for sorted order.
```

Comment:
`number[]` is wrong. Any comparable objects are fine. (Strings would be fine as well, for example, though the order may not be what you expect.)

The comments are also a bit redundant. You're effectively stating the same thing three times.